src/dbos-executor.ts

```diff
@@ -383,6 +383,7 @@ export class DBOSExecutor {
       authenticatedRoles: wCtxt.authenticatedRoles,
       request: wCtxt.request,
       executorID: wCtxt.executorID,
+      createdAt: Date.now() // Remember the start time of this workflow
```

Comment:
What's the format of this?

Reply:
This is equivalent to the previous one in PG. Epoch in milliseconds in UTC.